guide/glossary.md

```diff
@@ -251,19 +251,37 @@ A standard for multi-signature that uses Schnorr signatures. Previously, the mor
 
 ### Node
 
-Node refers to [software]({{ '/guide/getting-started/software/#nodes' | relative_url }}) that participates in the bitcoin network. It exchanges transaction data with other nodes, stores some or all of it, and verifies that transactions are valid. There is also dedicated [node hardware]({{ '/guide/getting-started/hardware/#hardware-wallets' | relative_url }}).
+A node refers to a Bitcoin client that participates in the Bitcoin network.
 
-#### Full node
+#### Full nodes
 
-A node that fully verifies all transactions and rules of the Bitcoin network.
+Full nodes are Bitcoin clients that fully verify and enforce the rules of the Bitcoin network. An example of a rule is that there will never be more than 21 million bitcoin.
 
-#### Pruned full node
+##### Archival full node
 
-A node that fully verifies all of the rules of the Bitcoin network with a much smaller storage capacity because it only stores transactions relevant to the users wallet.
+A full node that stores the entire Bitcoin block chain, as well as sends historical data to other nodes.
 
-#### Archival node
+##### Pruned full node
 
-A full node that has not only verified all data, but also stores and provides it to other nodes on the network.
+A full node that uses much lower storage capacity as it only stores a portion of recent blocks. 
+
+#### Light nodes
+
+Light nodes are Bitcoin clients that do not verify or enforce the rules of the Bitcoin network.
```

Comment:
Reading this description makes me ask what the point of light nodes is if they don't verify or enforce rules. What are they good at/for?

Reply:
This first description explains it in a general, low level whilst the SPV / Neutrino entries give examples as to what they are good for